release-notes/5.0/5.0-supported-os.md

```diff
@@ -33,7 +33,7 @@ Red Hat Enterprise Linux      | 7+                            | x64            |
 Red Hat Enterprise Linux <br> CentOS <br> Oracle Linux | 7+    | x64            | [Red Hat support policy](https://access.redhat.com/support/policy/updates/errata/) <br> [CentOS lifecycle](https://wiki.centos.org/FAQ/General#head-fe8a0be91ee3e7dea812e8694491e1dde5b75e6d) <br> [Oracle Linux lifecycle](http://www.oracle.com/us/support/library/elsp-lifetime-069338.pdf)
 Fedora                        | 30+                    | x64            | [Fedora lifecycle](https://fedoraproject.org/wiki/End_of_life)
 Debian                        | 9+                       | x64, ARM32, ARM64     | [Debian lifecycle](https://wiki.debian.org/DebianReleases)
-Ubuntu                        | 16.04+                  | x64, ARM32, ARM64   | [Ubuntu lifecycle](https://wiki.ubuntu.com/Releases)
+Ubuntu                        | 19.10, 18.04, 16.04                | x64, ARM32, ARM64   | [Ubuntu lifecycle](https://wiki.ubuntu.com/Releases)
```

Comment:
No need to update the Out of support section with 19.04 as that was never part of support matrix.